test/actions/Test_Actions.php

```diff
@@ -1498,5 +1498,46 @@ public function cannot_set_empty_actionid()
         // An empty ActionID
         $action->setActionID('');
     }
+
+    /**
+     * @test
+     * @expectedException \PAMI\Exception\PAMIException
+     */
+    public function cannot_set_updateconfig()
```

Comment:
I'm not sure I understand what this test is about.. on one hand, an UpdateConfigAction is being created, but a Logoff action is being sent, so the test will fail (but the exception is just an Exception instead of a PAMIException, so that's one thing).

We should be testing the UpdateConfigAction "in the positive case" (all the can_* methods), where we setup all the arguments and assert that it gets written correctly to the socket instead of asserting that with some sort of negative case with a LogoffAction.